### ISSUE TITLE
Get server and client certificate info

### ISSUE THREAD

**Issue by victorthoang** [MicrosoftEdge/WebView2Feedback#2798]
### Discussed in https://github.com/MicrosoftEdge/WebView2Feedback/discussions/2665

<div type='discussions-op-text'>

<sup>Originally posted by **bostjane** August 10, 2022</sup>
I would like to see the functionality that enables me to get server and client certificate info. Currently there is only API to select user certificate and handle server certificate error. However, if server certificate is fine and webview2 connects to it and/or user selects client certificate by himself, application has no way of getting info for server and client certificate.

Would very like to see this functionality in further releases...</div>

[AB#41355902](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/41355902)

**Comment by nishitha-burman** [comment]
Hi @bostjane can you please provide details on the scenario you would like to support with this feature?

Thanks!

**Comment by nishitha-burman** [comment]
Looks like this feature request is being tracked in #2602, closing this issue and tracking request in #2602. Please re-open if this is for a different feature request. Thanks!

**Comment by bostjane** [comment]
Suggested solution to get client certificate works only partially. I know it is possible to handle client request events, but in case I want to use build in UI for certificate selection in the event, then host app does not know which certificate user selected in UI.
The only way I know to determine which certificate was chosen is to use/build custom certificate selection UI. Is that right? If so, I would miss a common way to determine used client certificate, no matter which way was selected.

Thanks.

**Comment by monica-ch** [comment]
@bostjane That's right, currently app has to handle the event and build a custom ui to know which client certificate has been selected by the user. Can you use it as a workaround until we build an API for this.
